packages/did-core-test-server/suites/did-core-properties/did-core-properties.js

```diff
@@ -298,7 +298,7 @@ const generateDidCorePropertiesTests = (
           } else if(Array.isArray(serviceValue.type)) {
             const types = serviceValue.type;
             types.forEach(type => {
-              expect(typeof type === 'string').toBe(true);
```

Comment:
This can be replaced with `isInfraString` but looks like I failed to export the predicate. Will look into my code.

Reply:
No, not here. different `if`.